packages/core/src/types/config.ts

```diff
@@ -65,6 +65,7 @@ export interface MarkConfiguration {
   environment: Environment;
   logLevel: LogLevel;
   supportedSettlementDomains: number[];
+  prioritizeOldestInvoice?: boolean;
```

Comment:
is the only way to change this a config update? need to make sure that Andrew/khalil are aware of how to tweak this preference

Reply:
Not atm, Andrew said we should keep this false for now so we'll figure a different way to switch down the line.

Reply:
Okay that will lead to problems if we ever have one large invoice but we can work around it